src/middlewared/middlewared/etc_files/local/collectd.conf

```diff
@@ -247,10 +136,15 @@ LoadPlugin nfsstat
 </Plugin>
 
 <Plugin "df">
-	Mountpoint "/"
-	Mountpoint "^\/mnt(?:(?!\.zfs\/snapshot).)*$"
-	FSType "zfs"
-	LogOnce true
+	Mountpoint "/^\/boot/"
+	Mountpoint "/\.zfs\/snapshot/"
+	Mountpoint "/\.system/"
+	Mountpoint "/\.warden/"
+	FSType "tmpfs"
+	FSType "nullfs"
+	FSType "devfs"
+	FSType "fdescfs"
+	ignoreSelected true
```

Comment:
Why is `LogOnce true` gone?

Reply:
This was to reflect the latest changes introduced by this commit - https://github.com/freenas/freenas/commit/b1e0feb9a6c9af732b3abbfa5527cc305bc560e6#diff-f179f9839486600944714a1c9a32c87a

Reply:
That commit was wrong, we need to keep LogOnce

Reply:
I'll update the PR asap